installing/installing_aws/installing-aws-outposts.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-aws-outposts"]
+= Installing a cluster on AWS with compute nodes on AWS Outposts
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-outposts
+
+toc::[]
+
+In {product-title} version 4.14, you could install a cluster on
+Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview.
+As of {product-title} version {product-version}, this installation method is no longer supported.
+Instead, you can
+xref:../../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[install a cluster on AWS into an existing VPC]
+and provision compute nodes on AWS Outposts as a postinstallation configuration task.
+
+//For more information, see PUT AN XREF HERE SOON
```

Comment:
sgtm. Will we wait for the PR  https://github.com/openshift/openshift-docs/pull/71263 to merge before this?

Reply:
Yes, that's where the link to Jeana's content will go :)

Reply:
https://github.com/openshift/openshift-docs/pull/71263/files